trade-finance-manager-ui/server/controllers/utilisation-reports/add-to-an-existing-payment/index.ts

```diff
@@ -0,0 +1,45 @@
+import { Response } from 'express';
+import { getFormattedReportPeriodWithLongMonth } from '@ukef/dtfs2-common';
+import { AddToAnExistingPaymentViewModel } from '../../../types/view-models';
+import api from '../../../api';
+import { asUserSession } from '../../../helpers/express-session';
+import {
+  getFeeRecordIdsFromPremiumPaymentsCheckboxIds,
+  getPremiumPaymentsCheckboxIdsFromObjectKeys,
+} from '../../../helpers/premium-payments-table-checkbox-id-helper';
+import { CustomExpressRequest } from '../../../types/custom-express-request';
+import { PRIMARY_NAVIGATION_KEYS } from '../../../constants';
+import { mapToSelectedReportedFeesDetailsViewModel, PremiumPaymentsTableCheckboxSelectionsRequestBody } from '../helpers';
+
+type AddToAnExistingPaymentRequest = CustomExpressRequest<{
+  reqBody: PremiumPaymentsTableCheckboxSelectionsRequestBody;
+}>;
+
+const renderAddToAnExistingPaymentPage = (res: Response, context: AddToAnExistingPaymentViewModel) =>
+  res.render('utilisation-reports/add-to-an-existing-payment.njk', context);
+
+export const addToAnExistingPayment = async (req: AddToAnExistingPaymentRequest, res: Response) => {
+  try {
+    const { user, userToken } = asUserSession(req.session);
+    const { reportId } = req.params;
+    const checkedCheckboxIds = getPremiumPaymentsCheckboxIdsFromObjectKeys(req.body);
+    const feeRecordIds = getFeeRecordIdsFromPremiumPaymentsCheckboxIds(checkedCheckboxIds);
+
+    // TODO FN-1749: Handle form submission.
```

Comment:
I think there is more of a chance of forgetting about this TODO then there is of forgetting to handle the form submission in the future ticket so I'd prefer it if we didn't merge this in.